src/eei.cpp

```diff
@@ -133,6 +133,25 @@ Literal EthereumInterface::callImport(Import *import, LiteralList& arguments) {
       return Literal((uint32_t)code.size());
     }
 
+    if (import->base == Name("externalCodeCopy")) {
+      cout << "externalcodecopy" << endl;
+
+      uint32_t addressOffset = arguments[0].geti32();
+      uint32_t resultOffset = arguments[1].geti32();
+      uint32_t codeOffset = arguments[2].geti32();
+      uint32_t length = arguments[3].geti32();
+
+      evm_address address = loadUint160(addressOffset);
+      const uint8_t *code;
+      size_t code_size = context->fn_table->get_code(&code, context, &address);
```

Comment:
@chfast `get_code` returns a pointer to the pointer, not clear who needs to free that